pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -225,7 +225,7 @@ func extractFiles(archiveFile string, pluginName string, filePath string, allowS
 			return fmt.Errorf("filepath: %q tries to write outside of plugin directory: %q, this can be a security risk",
 				zf.Name, filepath.Join(filePath, pluginName))
 		}
-		newFile := filepath.Join(filePath, newFileName)
+		newFile := filepath.Join(filepath.Clean(filePath), newFileName)
```

Comment:
I believe that the linter false positively complains for that given that `filepath.Join()` calls the `filepath.Clean()`
https://golang.org/src/path/filepath/path.go#L211
https://golang.org/src/path/filepath/path_unix.go#L45

Reply:
@papagian Ah thanks for pointing that out! I never researched why `filepath.Clean` should fix this potential vulnerability, so I lack a real understanding of the issue. @bergquist Can you elaborate on why `filepath.Clean` should help?

Reply:
@papagian What is your suggested resolution, that we put a comment `// nolint:gosec` on these lines?

Reply:
Either adding the comment until this is fixed securego/gosec#439
Otherwise keeping this change shouldn't do any harm (I just wanted to bring this up)

Reply:
@papagian Reading that issue thread, it looks as if `filepath.Clean` doesn't really solve anything wrt. this potential flaw.

Reply:
I came to the same conclusion

Reply:
@aknuds1 it was just a recommendation I saw in some issue.

Reply:
@bergquist I see it stops gosec from complaining, but I don't think it actually fixes anything (and the gosec issue pasted earlier seems to confirm my suspicion).

Reply:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved at initialization time.